src/ReturnTypes/StorageDynamicStaticMethodReturnTypeExtension.php

```diff
@@ -13,6 +13,7 @@
 use PHPStan\Type\DynamicStaticMethodReturnTypeExtension;
 use PHPStan\Type\ObjectType;
 use PHPStan\Type\Type;
+use function in_array;
```

Comment:
code sniffer will fail, this needs a newline

Reply:
Ok

Reply:
It has a new line
<img width="556" alt="image" src="https://github.com/user-attachments/assets/13802fa0-ffd4-416d-b55e-5f8aef2ccb80">

Reply:
Can you clarify more, please?

Reply:
The import groups (classes, functions, constants) need to be separated by a newline

```suggestion

use function in_array;
```

Reply:
I added space before it https://github.com/larastan/larastan/pull/2020/commits/d3abaaab1c931065f84649bf277b3b75be3b7bd1 
Thanks